### PR TITLE
Update the quickstart sample to be in line with other languages

### PR DESCRIPTION
**Pull request by JesseLovelace**
Updates the quickstart sample to create and run a minimal transfer job, also changes the region tag to use what the other languages will use

### CODE REVIEW

**Review by frankyn — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "prelint": "cd samples; npm link ../; npm i"
   },
   "dependencies": {
+    "@google-cloud/storage-transfer": "^1.1.1",
```
Comment (danielduhh) [inline package.json:42]: Is this on purpose -- the transfer package depending on itself?

**Review by danielduhh — COMMENTED** [review]
Looks solid just had a question about the added dependency

**Review by JustinBeckwith — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (danielduhh) [inline package.json:42]:
```suggestion
```

**Review by danielduhh — APPROVED** [review]

**Pull request merged**